schema/artist/index.js

```diff
@@ -18,9 +20,6 @@ import {
 let ArtistType = new GraphQLObjectType({
   name: 'Artist',
   fields: () => {
-    let Artwork = require('../artwork').default;
-    let PartnerShow = require('../partner_show');
-
```

Comment:
No more deferred requires needed. Looks like the circular dependencies issue was fixed in 6.